operator/pkg/gateway-api/routechecks/route_checks.go

```diff
@@ -93,6 +142,25 @@ func CheckBackendIsExistingService(input Input) (bool, error) {
 					Reason:  string(gatewayv1.RouteReasonBackendNotFound),
 					Message: err.Error(),
 				})
+				continue
+			}
+			// ServiceImport gateway api support is conditioned by the fact
+			// that an actual Service backs it. Other implementations of MCS API
+			// are not supported.
+			svcName, ok := svcImport.Annotations[mcsapicontrollers.DerivedServiceAnnotation]
+			if !ok {
+				input.SetAllParentCondition(metav1.Condition{
+					Type:    string(gatewayv1.RouteConditionResolvedRefs),
+					Status:  metav1.ConditionFalse,
+					Reason:  string(gatewayv1.RouteReasonBackendNotFound),
+					Message: fmt.Sprintf("%s %s/%s do not have annotation %s", svcImport.Kind, svcImport.Name, ns, mcsapicontrollers.DerivedServiceAnnotation),
```

Comment:
Nits:
- `svcImport.Name` and `ns` should be switched - this will read as `svcName/namespace`, which is the opposite of the usual rendering.
- `do not have annotation` should be `does not have annotation`. English sucks.